### ISSUE TITLE
Not finding installed dependencies

### ISSUE THREAD

**Issue by BellsToll** [lanmaster53/recon-ng#24]
Great software! Very advanced and comprehensive.

Having trouble with v5 finding dependencies that I install. For example account_creds module requires the pyaes. Therefore, i pip install pyaes, verify that the library is installed, and yet when v5 loads up it says accounts_creds is disabled because it cannot find pyaes. Run into the same issue with PyPDF3 and metacrawler module.  "modules reload" command will output the same dependency not found error.

What am I doing wrong?

Thank you

**Comment by lanmaster53** [comment]
This is usually the result of a confusion between where you are installing the dependencies, and the environment under which Recon-ng is running. pip vs. pip2 vs. pip3, etc. I've seen several folks run into this and for each one it was an environmental issue. Try hitting folks up in the Recon-ng slack if this isn't enough to get you down the right path.

**Comment by Outermarker01** [comment]
I have the same issue. Lanmaster53 could you please translate "Try hitting folks up in the Recon-ng slack if this isn't enough to get you down the right path" in plain English?

**Comment by lanmaster53** [comment]
1. Join the Recon-ng Slack channel.
2. Ask the people that are there.

Sounds like an environmental issue though.
